### ISSUE TITLE
Feature: Ability to group similar vimbits and flag duplicates

### ISSUE THREAD

**Issue by darcyparker** [kkuchta/Vimbits#15]
There's lots of cool vimbits, but I am finding many are similar. Some are improvements over the earlier vimbits. While others are duplicates.  It would be nice if there was a way to propose consolidating the duplicates and also link to the similar and potentially improved/better vimbits.

**Comment by kkuchta** [comment]
Yep- this definitely needs to happen, as part of a few features to prevent vimbits from becoming a huge pile of 1-vote submissions where only the newest and the top few ever get seen.  Duplication flagging/merging, a 'Hot' list, and better tag exploration are currently on the todo list.
